pkg/xlogger/logger.go

```diff
@@ -2,6 +2,8 @@ package xlogger
 
 import "go.uber.org/zap"
 
+// Logger is a logger that can be used to log messages.
+// Deprecated: use https://pkg.go.dev/go.uber.org/zap@v1.24.0/zap instead.
```

Comment:
Que ocurre si un día cambiamos de zap a otro logger?

Reply:
No suele ocurrir, pero de última es un find and replace (si es que mantiene la compatibilidad).

Yo lo había agregado para poder mockear el log en unos tests, pero descubrí que el zap ya tiene soporte para tests, con lo cual es innecesario. Además había una interfaz para eso, pero seguía habiendo una dependendencia en los `zap.Fields`. Con lo cual ya no hay ganancia en introducir esa complejidad.